Sources/Rendering/Core/Renderer/index.js

```diff
@@ -284,7 +284,7 @@ function vtkRenderer(publicAPI, model) {
 
     if (nothingVisible) {
       vtkMath.uninitializeBounds(allBounds);
-      vtkDebugMacro('Can\'t compute bounds, no 3D props are visible');
+     // vtkDebugMacro('Can\'t compute bounds, no 3D props are visible');
```

Comment:
Do we want to keep them? Maybe we can remove the logging of debug by default instead?